docs/app/pages/components/sections/select/select/snippets/app.html

```diff
@@ -1,5 +1,5 @@
-<div class="row">
-    <div class="col-sm-6 col-xs-12">
+<div class="row m-t-xl">
+    <div class="col-sm-6 col-xs-12 m-t-xl">
```

Comment:
Presumably this is to allow dropDirection="up", in which case the appropriate change is to add a CSS class with margin-top of 250px, matching the dropdown height.